roles/ceph-common/handlers/main.yml

```diff
@@ -2,41 +2,19 @@
 - name: update apt cache
   apt:
     update-cache: yes
+  when: ansible_os_family == 'Debian'
 
 - name: restart ceph mons
-  service:
-      name: ceph-mon@{{ monitor_name }}
-      state: restarted
-  when:
-    - socket.rc == 0
-    - mon_group_name in group_names
+  include: "{{ playbook_dir }}/roles/ceph-common/handlers/restart-mon.yml"
```

Comment:
Is it necessary to use ``{{ playbook_dir }}`` here? Seems like it could just be ``./restart-mon.yml`` couldn't it?